index.html

```diff
@@ -245,6 +245,18 @@ <h2 class="contact-title">I'm always interested in hearing<span class="newline">
             <img src="assets/images/contact-frame3.svg" id="contact-frame3" alt="contact-frame3">
         </div>
     </section>
+    <section id="mobile-menu">
+        <div class="icon-wrapper">
+            <img src="assets/icons/close-menu-icon.svg" alt="close menu icon">
+        </div>
+        <div class="menu-list">
+            <ul>
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
```

Comment:
- [ ] These should be links that take users to corresponding part of the page. Kindly add links 😉

Reply:
Got it 👌🏽